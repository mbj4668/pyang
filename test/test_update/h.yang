module h {
  namespace urn:h;
  prefix h;

  import g {
    prefix g;
  }

  revision 2014-03-01;

  grouping g1 {
    container foo;
  }

  grouping g2 {
    container bar;
  }

  augment /g:a {
    uses g1;
  }

  augment /g:a {
    uses g2;
  }

  augment /g:b {
    uses g1;
  }

  augment /g:b {
    uses g2;
  }

  augment /g:c {
    uses g1;
  }

}
