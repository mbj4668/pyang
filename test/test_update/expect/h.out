h@2014-04-01.yang:1: error: CHK_DEF_REMOVED
h@2014-04-01.yang:1: error: CHK_DEF_REMOVED
h@2014-04-01.yang:1: error: CHK_DEF_REMOVED
